<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>982</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>831</width>
     <height>901</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_applte">
    <attribute name="title">
     <string>Approximations and LTE</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>162</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_buttons_applte">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QFormLayout" name="fl_le_applte">
        <item row="0" column="0">
         <widget class="QLabel" name="label_x0">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;x&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="le_x0">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_X">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="le_X">
          <property name="text">
           <string>60</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_y0">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;y&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="le_y0">
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_N">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>N</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="le_N_applte">
          <property name="text">
           <string>70</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_exact">
        <property name="text">
         <string>Exact Solution</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_eem">
        <property name="text">
         <string>Explicit Euler Method</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_iem">
        <property name="text">
         <string>Improved Euler Method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_rkm">
        <property name="text">
         <string>Runge-Kutta Method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_eem_lte">
        <property name="text">
         <string>EEM LTE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_iem_lte">
        <property name="text">
         <string>IEM LTE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_rkm_lte">
        <property name="text">
         <string>RKM LTE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_plot_applte">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>611</width>
       <height>821</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_graphs">
      <item>
       <widget class="QGroupBox" name="gb_app">
        <property name="title">
         <string>Exact solution and approximation graphs</string>
        </property>
        <widget class="QWidget" name="app_plot" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>581</width>
           <height>371</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gb_lte">
        <property name="title">
         <string>Local truncation error graphs</string>
        </property>
        <widget class="QWidget" name="lte_plot" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>581</width>
           <height>331</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_applte">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>160</width>
       <height>121</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plotted successfully&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_applte_colors">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;Exact Solution - blue&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Explicit Euler Method - black&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Improved Euler Method - red&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#00aa00;&quot;&gt;Runge-Kutta Method - green&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_gte">
    <attribute name="title">
     <string>GTE</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>162</width>
       <height>194</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vl_buttons_gte">
      <item>
       <layout class="QFormLayout" name="fl_le_gte">
        <item row="0" column="0">
         <widget class="QLabel" name="label_n0">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;n&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="le_n0">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_N_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>N</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="le_N_gte">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_eem_gte">
        <property name="text">
         <string>Explicit Euler Method</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_iem_gte">
        <property name="text">
         <string>Improved Euler Method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_rkm_gte">
        <property name="text">
         <string>Runge-Kutta Method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_plot_gte">
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="gb_gte">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>609</width>
       <height>366</height>
      </rect>
     </property>
     <property name="title">
      <string>Global truncation error graphs</string>
     </property>
     <widget class="QWidget" name="gte_plot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>581</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_gte">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>160</width>
       <height>141</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plotted successfully&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_gte_colors">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;Exact Solution - blue&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Explicit Euler Method - black&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Improved Euler Method - red&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#00aa00;&quot;&gt;Runge-Kutta Method - green&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;F21. Differential Equations. Computational Practicum&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
